app/models/manageiq/providers/kubevirt/infra_manager/event_parser.rb

```diff
@@ -0,0 +1,25 @@
+class ManageIQ::Providers::Kubevirt::InfraManager::EventParser
+  def self.event_to_hash(event, ems_id = nil, source)
```

Comment:
Added `source` as a parameter to differentiate between Kubevirt and OSV. Since OSV will be using this same parser class